File: samples/java_layout/src/main/java/io/customer/android/sample/java_layout/ui/inline/MultipleInlineFragment.java
package io.customer.android.sample.java_layout.ui.inline;

import androidx.annotation.NonNull;

import io.customer.android.sample.java_layout.databinding.FragmentMultipleInlineBinding;
import io.customer.android.sample.java_layout.ui.core.BaseFragment;

public class MultipleInlineFragment extends BaseFragment<FragmentMultipleInlineBinding> {
    
    private static final String TAG = "MultipleInlineFragment";

    @NonNull
    @Override
    protected FragmentMultipleInlineBinding inflateViewBinding() {
        return FragmentMultipleInlineBinding.inflate(getLayoutInflater());
    }

    @Override
    protected void setupContent() {
        // Set action listeners for all inline in-app messages
        binding.firstInappMessage.setActionListener(new InlineMessageActionListenerImpl(requireContext(), "Multi Top"));
        binding.secondInappMessage.setActionListener(new InlineMessageActionListenerImpl(requireContext(), "Multi Center"));
        binding.thirdInappMessage.setActionListener(new InlineMessageActionListenerImpl(requireContext(), "Multi Bottom"));
    }

    public static MultipleInlineFragment newInstance() {
        return new MultipleInlineFragment();
    }
}

File: samples/kotlin_compose/src/main/java/io/customer/android/sample/kotlin_compose/ui/inline/components/MultipleInlineFragment.kt
package io.customer.android.sample.kotlin_compose.ui.inline.components

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import io.customer.android.sample.kotlin_compose.databinding.FragmentMultipleInlineBinding

class MultipleInlineFragment : Fragment() {
    private var _binding: FragmentMultipleInlineBinding? = null
    private val binding get() = _binding!!

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        _binding = FragmentMultipleInlineBinding.inflate(inflater, container, false)
        return binding.root
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }

    companion object {
        private const val TAG = "MultipleInlineFragment"

        @JvmStatic
        fun newInstance() = MultipleInlineFragment()
    }
}
